docs/docs/index.md

```diff
@@ -4,27 +4,12 @@ ms.date: 09/30/2019
 ---
 
 # Factory Orchestrator
+Factory Orchestrator, is a cross-platform service for organizing, executing, and logging a set of executable scripts, binaries, or ["Tasks"](tasks-and-tasklists.md) on a system. Factory Orchestrator tracks task information, including run unique per-run results and logging; even persisting task state to allow the service to be resilient to data loss due to system failure. Factory Orchestrator also provides an optional [UWP client app](use-the-factory-orchestrator-app.md) and [robust client API surface](use-the-factory-orchestrator-api.md) for clients to monitor & interact with the service via the App or C# .NET, C# UWP, or PowerShell code. The UWP app and and any other client can communicate with any Factory Orchestrator service running on the same system and/or over a network!
 
-Factory Orchestrator provides a simple and reliable way to run and manage factory line validation and fault analysis workflows. Beyond the factory floor, Factory Orchestrator can be used during OS and hardware development to support various developer inner-loop and diagnostics activities.
-
-Factory Orchestrator consists of two components:
 
-- A .NET Core system service (Microsoft.FactoryOrchestrator.Service.exe): The service tracks task information, including run unique per-run results and logging; even persisting task state to allow the service to be resilient to data loss due to client failure. It also provides a robust API surface for clients to monitor & interact with the service via C# .NET, C# UWP, or PowerShell code. 
+Factory Orchestrator provides a simple and reliable way to run and manage factory line validation and fault analysis workflows. Beyond the factory floor, Factory Orchestrator can be used during OS and hardware development to support various developer inner-loop and diagnostics activities.

```

Comment:
slight re-write:

"Factory Orchestrator provides a simple and reliable way to run validation or calibration software on target devices on a manufacturing line. Factory Orchestrator makes auditing and fault analysis by providing durable and easily identifiable 'per run' results and logs. Beyond the factory floor, Factory Orchestrator can be used during device development to support various developer inner-loop and diagnostics activities enabling further reuse of collateral created and leveraged across the full device manufacturing lifecycle.' 

feel free to re adjust as well. :)

Reply:
can also use 'device under test' as it seems like we could link to wikipedia/its known in public domain: https://en.wikipedia.org/wiki/Device_under_test